### PR TITLE
Update monitoring page

### PR DESCRIPTION
**Pull request by mooreds**
Added in high level overview table.

Also changed headings.

### CODE REVIEW

File: site/docs/v1/tech/installation-guide/monitor.adoc [hunk 1]
```diff
@@ -6,27 +6,78 @@ description: Understand how your installation is operating by monitoring it.
 
 == Overview
 
-You can monitor FusionAuth in a number of ways.
+Once you've installed and integrated FusionAuth, you have to operate it. And for that you need insight into how the system is performing. There are a number of ways to do so.
```
Comment (jobannon) [inline site/docs/v1/tech/installation-guide/monitor.adoc:9]:
```
Once you've installed and integrated FusionAuth, you have to operate it. 
```

This sounds odd to me.

File: site/docs/v1/tech/installation-guide/monitor.adoc [hunk 2]
```diff
@@ -6,27 +6,78 @@ description: Understand how your installation is operating by monitoring it.
...
+* <<Metrics>>
+** <<Prometheus Endpoint>>
+* <<Load Testing>>
+** <<Tips>>
+
+== Monitoring Summary Table
+
+Here's an overview table of monitoring options.
+
+.Monitoring Overview
```
Comment (jobannon) [inline site/docs/v1/tech/installation-guide/monitor.adoc:25]: 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 

This is an awesome idea!  Thanks for writing it!

💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯

File: site/docs/v1/tech/installation-guide/cloud.adoc [hunk 3]
```diff
@@ -481,3 +481,4 @@ FusionAuth Cloud has the same link:/docs/v1/tech/reference/limitations/[limitati
...
 * If you are on FusionAuth Cloud and you find that some requests are failing, it is possible you are being rate limited. This isn't intentional, but an automated part of our infrastructure to ensure FusionAuth Cloud performance and security. If you are rate limited but need these requests to occur, please open a https://account.fusionauth.io/acccount/support/[support ticket] with details.
 * If you want to run Advanced Threat Detection, an Enterprise feature, you'll need a cloud deployment with sufficient memory. Currently that means it must be a Large or X-Large size.
+* With HA deployments, requests to public metric endpoints such as `/api/status` are round-robined to individual servers and not aggregated across the deployment.
```
Comment (jobannon) [inline site/docs/v1/tech/installation-guide/cloud.adoc:484]: Maybe .... So make sure that you check both nodes!

**Review by jobannon — APPROVED** [review]
This is stellar - a very good addition to our doc.

File: site/docs/v1/tech/installation-guide/cloud.adoc (hunk 3 above)
Comment (mooreds) [reply site/docs/v1/tech/installation-guide/cloud.adoc:484]: No way for them to do so, unfortunately.

Comment (mooreds) [reply site/docs/v1/tech/installation-guide/cloud.adoc:484]: We don't expose/document individual instances.

**Pull request merged**